cockatrice/src/dlg_connect.cpp

```diff
@@ -220,6 +205,11 @@ void DlgConnect::rebuildComboBoxList(int failure)
         }
     }
 
+    // If first run, set to RR
+    if (settingsCache->servers().getPrevioushostName().isEmpty()) {
+        previousHosts->setCurrentIndex(1);
```

Comment:
Are we able to guarantee this index exists? What if it fails to get the servers?

Reply:
Good point, will add a check